azurerm/resource_arm_storage_account.go

```diff
@@ -676,24 +747,43 @@ func resourceArmStorageAccountRead(d *schema.ResourceData, meta interface{}) err
 		if endpoints := props.SecondaryEndpoints; endpoints != nil {
 			if blob := endpoints.Blob; blob != nil {
 				d.Set("secondary_blob_endpoint", blob)
+				if v := endpoints.Blob; v != nil {
+					if u, err := url.Parse(*v); err != nil {
```

Comment:
We should either be checking for an error and bubbling it up or refactor this to not use `url.Parse()`